pkg/polaris/graphql/core/core.go

```diff
@@ -353,6 +353,49 @@ func (a API) WaitForTaskChain(ctx context.Context, id uuid.UUID, wait time.Durat
 	}
 }
 
+// WaitForFeatureDisableTaskChain waits for the feature disable task chain to
+// finish. If an error occurs while waiting for the task chain or the task chain
+// ends in a failed state, an error is returned.
+func (a API) WaitForFeatureDisableTaskChain(ctx context.Context, taskChainID uuid.UUID, featureStatus func(ctx context.Context) (bool, error)) error {
+	a.log.Print(log.Trace)
+
+	ctx, cancel := context.WithTimeout(ctx, 9*60*time.Second)
```

Comment:
nit - 9*time.Minute?

Reply:
LOL, I was looking for that using the Goland Intellisense feature, but couldn't find it. 😂